integration-cli/docker_cli_daemon_test.go

```diff
@@ -2238,3 +2238,28 @@ func (s *DockerDaemonSuite) TestDaemonLogOptions(c *check.C) {
 	c.Assert(err, check.IsNil, check.Commentf(out))
 	c.Assert(out, checker.Contains, "{json-file map[]}")
 }
+
+func (s *DockerDaemonSuite) TestDaemonRestartWithAutoRemoveContainer(c *check.C) {
```

Comment:
Yes, do you think we should run it on Windows? I notice that there's no daemon tests can run on Windows, though I don't know why. Guess some restriction from implementation on Windows?